SPECS/python-gevent/skip-irrelevant-tests.patch

```diff
@@ -0,0 +1,116 @@
+From 752bb897fb8abe93f6d023765c45595934ace78f Mon Sep 17 00:00:00 2001
+From: Nick Samson <nisamson@microsoft.com>
+Date: Tue, 2 Jul 2024 15:50:39 -0700
+Subject: [PATCH] Removed flaky/inapplicable tests for Azure Linux
+
+---
+ src/gevent/tests/test__core_timer.py | 59 +---------------------------
+ src/gevent/tests/test__socket_dns.py |  8 +---
+ src/greentest/3.12/test_socket.py    |  1 +
+ 3 files changed, 5 insertions(+), 63 deletions(-)
+
+diff --git a/src/gevent/tests/test__core_timer.py b/src/gevent/tests/test__core_timer.py
+index 97caa754..8e0b691f 100644
+--- a/src/gevent/tests/test__core_timer.py
++++ b/src/gevent/tests/test__core_timer.py
+@@ -95,63 +95,8 @@ class TestTimerResolution(Test):
+     # 0.02 or higher.
+     @reraises_flaky_timeout(AssertionError)
+     def test_resolution(self): # pylint:disable=too-many-locals
```

Comment:
Is there are more central place that we could patch instead?  Somewhere we can just remove the invocatio of the test?

Reply:
Not trivially; the tests are [discovered](https://github.com/gevent/gevent/blob/master/tox.ini) through a single invocation. Additionally, some of the tests are dynamically generated/invoked, meaning the only way to disable them properly is to either disable the entire suite (which is overly broad) or to bypass them in the source, like in this patch.

Reply:
Here's an example of [the generated tests.](https://github.com/microsoft/azurelinux/pull/9549/files/4acfb88b2c7435aa48ee8fc3bd9501bedaa14f87#diff-cdd57ffc0117ea266a8b21e69572744a26b095d1b9e6fd37044027f5a4e0203eR90-R100)